spec/services/indexer_spec.rb

```diff
@@ -162,7 +163,7 @@
         it 'has required fields' do
           expect(solr_doc).to include('milestones_ssim', 'released_to_ssim', 'wf_ssim', 'tag_ssim')
 
-          expect(solr_doc['originInfo_date_created_tesim']).to eq ['2021-01-01']
```

Comment:
I'm confused about why this expectation is changing? Doesn't the integration test expect YYYY-MM-DD?

Reply:
thanks for spotting this -- the publication date is a year (it's a facet) while this date is ... a date.  :-)   Fixed.